lib/course_students.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class CourseStudents
```

Comment:
Rather than a separate class for creating an array of userpage names, I think this would make more sense as a method on CoursesUsers, so that you could call `.userpage`. We actually have some closely related methods already, spread across User and CoursesUsers, so it would make sense do some refactoring to minimize duplicated logic.

Reply:
okay